docs: Rename registration.entity to registration.entity_id (#985)
oasisprotocol/docs: docs/node/run-your-node/validator-node.mdx

oasis account entity init -o /localhostdir/entity/entity.json --account my_entity
```

Now, we can move on to configuring our Oasis node with the information from
freshly generated `entity.json`.

:::info

You can obtain your entity ID by running the `cat entity.json` command and
reading out the `id` field.

> Alternatively, if your entity account is imported into the [Oasis CLI] named
> `my_entity` run the following command `oasis wallet show my_entity`. Your

Reviewer: ```suggestion
Alternatively, if your entity account is imported into the Oasis CLI you can use
the [`oasis wallet show`] command. Your
```

entity ID will be displayed in the `Public Key` field.

:::

[Ledger]: ../../general/manage-tokens/holding-rose-tokens/ledger-wallet.md
[offline/air-gapped machine]: https://en.wikipedia.org/wiki/Air_gap_\(networking\)
[`oasis wallet create`]: ../../general/manage-tokens/cli/wallet.md#create
[`oasis wallet import`]: ../../general/manage-tokens/cli/wallet.md#import
[`oasis wallet import-file`]: ../../general/manage-tokens/cli/wallet.md#import-file
[`oasis account entity init`]: ../../general/manage-tokens/cli/account.md#entity-init

> [Oasis CLI]: ../../general/manage-tokens/cli/README.md

Reviewer: ```suggestion
[`oasis wallet show`]: ../../general/manage-tokens/cli/wallet.md#show
```

oasis account entity init -o /localhostdir/entity/entity.json --account my_entity
```

Now, we can move on to configuring our Oasis node with the information from
freshly generated `entity.json`.

:::info

> You can obtain your entity ID by running the command: `cat entity.json`, or

Reviewer: ```suggestion
You can obtain your entity ID by running the `cat entity.json` command and
reading out the `id` field.
```

> alternatively, by using the Oasis CLI with the following command:
> `oasis wallet show my_entity`. If you use the Oasis CLI, your entity ID will be
> displayed in the `Public Key` field.

Reviewer: ````suggestion
Alternatively, if your entity account is imported into the [Oasis CLI] named `my_entity`
run the following command `oasis wallet show my_entity`. Your entity ID will be
displayed in the `Public Key` field.

[Oasis CLI]: ../../general/manage-tokens/cli/README.md
````

  You can find the current Oasis Seed Node address in the Network Parameters
  page ([Mainnet], [Testnet]).

> * `{{ node_entity_id }}`: The node's entity ID form `entity.json`.

Reviewer 2: ```suggestion
* `{{ node_entity_id }}`: The node's entity ID from `entity.json`.
```
